app/mailers/client_mailer.rb

```diff
@@ -9,4 +9,11 @@ def send_email(pdfkit, invoice, user)
     mail(to: @client.contact_details.first.email,
          subject: "[#{user.company.name}] Invoice #{invoice.invoice_number} of #{invoice.currency_code.upcase} #{invoice.total_amount} for #{@client.name}")
   end
+
+  def send_email_payment(client,invoice,user)
```

Comment:
You can always get client from invoice. No need to send it explicitly. 

There must be one space after `,` comma in any expression